packages/boot/test/bootstrapTests/test-orchestration.ts

```diff
@@ -167,9 +169,131 @@ test.serial('stakeAtom - smart wallet', async t => {
   t.like(wd.getLatestUpdateRecord(), {
     status: { id: 'request-account', numWantsSatisfied: 1 },
   });
-
-  const { ATOM } = agoricNamesRemotes.brand;
+  const { ATOM, DAI_axl, BLD } = agoricNamesRemotes.brand;
   ATOM || Fail`ATOM missing from agoricNames`;
+  DAI_axl || Fail`DAI_axl missing from agoricNames`;
+  BLD || Fail`BLD missing from agoricNames`;
+
+  // 2. Deposit to Account
+  await wd.executeOffer({
+    id: 'request-deposit-success',
+    invitationSpec: {
+      source: 'continuing',
+      previousOffer: 'request-account',
+      invitationMakerName: 'Deposit',
+    },
+    proposal: {
+      give: {
+        // @ts-expect-error BoardRemote is not assignable to Brand<any>
+        ATOM: { brand: ATOM, value: 100n },
+      },
+      exit: { waived: null },
+    },
+  });
+  t.like(wd.getLatestUpdateRecord(), {
+    status: { id: 'request-deposit-success', numWantsSatisfied: 1 },
+  });
```

Comment:
Hm. I'd like to see a check that the money got there. Can we capture calls across the bridge and check for a suitable one here?

Reply:
The best we can do in this environment is mock, which doesn't give the highest confidence or seem worth the cost.  I am going to pursue better unit testing at the exo level per your suggestion to get more confidence around these failure cases